WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -411,36 +411,45 @@ private extension GetStartedViewController {
             self.displayError(error as NSError, sourceTag: self.sourceTag)
         }
     }
+    
+    // MARK: - Send email
 
-    /// Offer host apps the opportunity to decide if they want to allow signups
+    /// Makes the call to request a magic signup link be emailed to the user.
     ///
-    func showSignupViewIfNecessary(_ error: Error) {
```

Comment:
This would completely remove the possibility for WooCommerce to allow signups, which I think is something that we want, but I am not completely sure it.

I will chat with Garance.

Reply:
>This would completely remove the possibility for WooCommerce to allow signups
Why? This function is just showing that view that I removed if Im not mistaken. The signup process remains the same minus a screen.

Reply:
Ouch, sorry! I misread the code. I, for some reason, thought that the delegate method to ask host apps if they want to support account signups had been removed too. Go figure.